logic/standalone/waterfall.py

```diff
@@ -0,0 +1,190 @@
+'''
+Logic module that can create waterfall objects.
+
+
+USAGE EXAMPLE:
+
+'''
+
+import logic.common.log_utils as log
+import logic.common.tiled_utils as tiled_utils
+
+#--------------------------------------------------#
+'''Variables'''
+
+# Template settings
+OBJECT_SUFFIX_FG = " fg"
+OBJECT_SUFFIX_BG = " bg"
```

Comment:
I'm kinda getting annoyed with these variables since it interrupts my train of thought to be reading code, see a variable I don't recognize and then scroll up to find what it is. Some uses cases of it are justified, but it's getting excessive IMO. Let's remove these 2 variables for now. In the future, I'll have the intern lint all the code files and get them up to a style I'd prefer (more on that later)